build_resolvers/lib/src/resolver.dart

```diff
@@ -304,23 +305,31 @@ List<int> _buildSdkSummary(String dartSdkPath) {
 /// [sdk].
 void _addFlutterLibraries(AbstractDartSdk sdk,
     ResourceProvider resourceProvider, String dartSdkPath) {
```

Comment:
I think `dartSdkPath` is unused here now :)

Reply:
Oops. Done.